Added an example of plotting a rectangle on a map with a rotation angle relative to the axes (#7348)
sunpy/sunpy: changelog/7348.feature.rst

> Add an example of plotting a rectangle on a map with a rotation angle relative to the axes :ref:`sphx_glr_generated_gallery_plotting_plot_rotated_rectangle.py`

Reviewer: ```suggestion
Add an example of plotting a rectangle on a map with a rotation angle relative to the axes (:ref:`sphx_glr_generated_gallery_plotting_plot_rotated_rectangle.py).`
```

> Add an example of plotting a rectangle on a map with a rotation angle relative to the axes (:ref:`sphx_glr_generated_gallery_plotting_plot_rotated_rectangle.py).`

Reviewer: ```suggestion
Add an example of plotting a rectangle on a map with a rotation angle relative to the axes (:ref:`sphx_glr_generated_gallery_plotting_plot_rotated_rectangle.py`).
```
Should fix:
`<unknown>:14: WARNING: undefined label: 'sphx_glr_generated_gallery_plotting_plot_rotated_rectangle.py).'

> Added an example of plotting a rectangle on a map with a rotation angle relative to the axes using :meth:`~sunpy.map.GenericMap.draw_quadrangle` using ``SkyOffsetFrame``

Reviewer: ```suggestion
Add an example of plotting a rectangle on a map with a rotation angle relative to the axes.
```
Can you also add the sphinx interlink to the gallery example as well?

Author: could you elaborate this please?

Reviewer: Hopefully https://sphinx-gallery.github.io/stable/auto_examples/plot_0_sin.html#cross-referencing helps
